ci/docker/runtime_functions.sh

```diff
@@ -542,7 +542,6 @@ build_ubuntu_cpu_clang60_mkldnn() {
     build_ccache_wrappers
 
     make \
-        ENABLE_TESTCOVERAGE=1         \
```

Comment:
Please don't do that.

Reply:
@marcoabreu `TESTCOVERAGE` leads to very large binary size resulting in linking error. What's the practical way can you recommend for now? We cannot just stop adding more operators.

Reply:
Agree. We'll investigate the linking error today.